app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -205,21 +214,13 @@ public void onConnected(Bundle bundle) {
     private void updateChapterPages() {
         final boolean wasOrganizer = mViewPagerAdapter != null
             && mViewPagerAdapter.getCount() == ORGANIZER_PAGES.length;
+        if (organizerCheckCallback != null) {
```

Comment:
It would be great to initialize this at the beginning with `wasOrganizer = false` to make sure that it is never null.